packages/core/src/fhir-to-cda/cda-templates/components/__tests__/xmls/encounters-section-single-entry.xml

```diff
@@ -53,6 +53,10 @@
 		</templateId>
 		<id root="placeholder-ORG-OID" extension="${encounterId}">
 		</id>
+		<code code="99214" codeSystem="2.16.840.1.113883.6.12">
+			<translation code="99214" codeSystem="2.16.840.1.113883.6.12" displayName="OV EST PT LEV 4">
+			</translation>
```

Comment:
also here. Weird to have a translation thats identical to the code. Probably same bug as with 207Q00000X?